packages/xo-server-backup-reports/templates/mjml/partials/vmText.hbs

```diff
@@ -1,11 +1,15 @@
-
 {{#if vm}}
 <h3>{{vm.name_label}}</h3>
-<ul>
-  <li><span style="font-weight: bold">UUID</span>: {{vm.uuid}}</li>
 {{else}}
 <h3>VM not found</h3>
+{{/if}}
+{{#each taskLog.infos}}
+  ℹ️ {{message}}
+{{/each}}
```

Comment:
We should likely display warnings as well.

Reply:
We actually already display warnings.
I'll adapt the way we display info so it looks more uniform